lib/drives/pages/drive_detail_page.dart

```diff
@@ -236,32 +236,7 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageDriveDetailTitle),
-        actions: _fullyLoaded
-            ? <Widget>[
-                IconButton(
-                  key: const Key('driveChatButton'),
-                  onPressed: () => Navigator.push(
-                    context,
-                    MaterialPageRoute<void>(
-                      builder: (BuildContext context) => DriveChatPage(
-                        drive: _drive!,
-                      ),
-                    ),
-                  ).then((_) => loadDrive()),
-                  icon: Badge(
-                    badgeContent: Text(
-                      _drive!.getUnreadMessagesCount().toString(),
-                      style: const TextStyle(color: Colors.white),
-                      textScaleFactor: 1.0,
-                    ),
-                    showBadge: _drive!.getUnreadMessagesCount() != 0,
-                    position: BadgePosition.topEnd(top: -12),
-                    child: const Icon(Icons.chat),
-                  ),
-                  tooltip: S.of(context).openChat,
-                ),
-              ]
-            : null,
+        actions: <Widget>[buildChatButton()],
```

Comment:
ah was that because of a merge conflict? I remember Simon and I built that method to handle a null drive

Reply:
yes i guess i messed that up while rebasing but now it's back again.